cmd/testworkflow-init/main.go

```diff
@@ -369,7 +376,18 @@ func main() {
 				stopTimeoutWatcher := orchestration.WatchTimeout(finalizeTimeout, leaf...)
 
 				// Run the command
-				commands.Run(*action.Execute, currentContainer)
+				d := data.GetState()
+				recorderConfig := utilization.Config{
+					Dir:       "./.tk/metrics",
+					Skip:      action.Execute.Toolkit,
+					Workflow:  d.InternalConfig.Workflow.Name,
+					Step:      step.Ref,
+					Execution: d.InternalConfig.Execution.Id,
+					Format:    core.FormatInflux,
+				}
+				utilization.WithMetricsRecorder(recorderConfig, func() {
```

Comment:
I don't see the metrics output being saved as artifact - is it meant to be uploaded only when users request it specifically?

Reply:
updated to use internal artifact storage